Controller/Adminhtml/CompanyCodes/Get.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * @category    ClassyLlama
+ * @copyright   Copyright (c) 2018 Classy Llama Studios, LLC
+ * @author      sean.templeton
+ */
+
+namespace ClassyLlama\AvaTax\Controller\Adminhtml\CompanyCodes;
+
+use ClassyLlama\AvaTax\Exception\AvataxConnectionException;
+use Magento\Framework\DataObject;
+
+class Get extends \Magento\Backend\App\Action
```

Comment:
@LordZardeck - Any backend controller needs to implement the _isAllowed method to specify the resource authorization.